### ISSUE TITLE
All my IOTA dissapeared from my wallet. It's seems like an ongoing problem for many people. IOTA should take this issue way more seriously and establish dedicated person to resolve this issue. My account was inactive for a couple months but that should not be a reason for my IOTA to dissapear. I generated my own seed printed and kept it safe. Those are out then, downloaded the newest version of the wallet, still nothing...now what? I would need someone please give me step by step instructions how to recover...something like instructions for dummies! Thank you 

### ISSUE THREAD

**Issue by 1dog2boys** [iotaledger/wallet#1222]
### Prerequisites
If you are **not** suggesting a feature, you must be able to check all of the following (place an x inside the brackets to check the box). If you cannot check all of the boxes, this issue should not be reported on GitHub as it is not a bug:

* [ ] I confirm that this is an issue with the IOTA Wallet and not an exchange
* [ ] I confirm that this is an issue with the wallet, not IRI
* [ ] For pending transactions: I confirm that I have tried reattaching and promoting multiple times (check this box if your issue is not related to a pending transaction)
* [ ] I confirm that this is not an issue related to stolen/lost funds
* [ ] I confirm that this is not a zero balance issue related to a snapshot (last snapshots occurred on October 23 and January 28)
* [ ] For private key warnings: I confirm that I am not at risk of reusing an address, or another user has verified that I am not (check this box if your issue is not related to a private key reuse)
* [ ] I confirm that this issue is present on the latest version of the wallet

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

[OS Version]
[Wallet Version]

**Comment by 1dog2boys** [comment]
All my IOTA dissapeared from my wallet. It's seems like an ongoing problem for many people. IOTA should take this issue way more seriously and establish dedicated person to resolve this issue. My account was inactive for a couple months but that should not be a reason for my IOTA to dissapear. I generated my own seed printed and kept it safe. Those are out then, downloaded the newest version of the wallet, still nothing...now what? I would need someone please give me step by step instructions how to recover...something like instructions for dummies! Thank you

**Comment by wonkytonky** [comment]
there's a snapshot going on

**Comment by 1dog2boys** [comment]
What that means a snapshot going on? Is that good or bad? New info: I generated 15 address and then I logged off but now 5 hours later I checked all the history (I mean that I generated those seeds) was gone so nothing in history at all...is that normal?

**Comment by lo5tinpr0** [comment]
Hey @1dog2boys , "snapshot" means that there is some kind of "update" going on on the IOTA network. Nothing you have to care about.
So, let's try this step by step:
At first, change your node to a working and up to date one, for example "http://nodes.iota.fm:80"
Should look like this:
![unbenannt](https://user-images.githubusercontent.com/37935152/39414911-336f7c5e-4c3d-11e8-92c9-47a2144b9b08.PNG)

After that, log in with your seed (could take some time) and have a look at your history. What can you see there? Could you upload a screenshot?

**Comment by 1dog2boys** [comment]
lo5tinpr0 
Hi, thanks for your reply
I tried several nodes, the current one is
 http://node02.iotatoken.nl:14265 seems like a good one.
"After that, log in with your seed (could take some time) and have a look at your history. What can you see there? Could you upload a screenshot?" 
As of now 96 transfers 86 addresses but interesting the history doesn't go to far the oldest is from 29/04/2018 nothing prior...I don't think this how it should operate. What happened with all my transfers when I sent the IOTA tokens from the trade sites to my wallet. I know they arrived I checked them. Also if I go to the trade places and see the history over there I can see the exact time and  addresses where the tokens been sent to (my address). So I wonder what's going on?  Where all my tokens taken into custody?
and I don't know how to make a screenshot but would be happy to learn it. 
Thank you

**Comment by 1dog2boys** [comment]
lo5tinpr0
Hi, thanks for your reply
I tried several nodes, the current one is
http://node02.iotatoken.nl:14265 seems like a good one.
"After that, log in with your seed (could take some time) and have a look at your history. What can you see there? Could you upload a screenshot?"
As of now 96 transfers 86 addresses but interesting the history doesn't go to far the oldest is from 29/04/2018 nothing prior...I don't think this how it should operate. What happened with all my transfers when I sent the IOTA tokens from the trade sites to my wallet. I know they arrived I checked them. Also if I go to the trade places and see the history over there I can see the exact time and addresses where the tokens been sent to (my address). So I wonder what's going on? Where all my tokens taken into custody?
and I don't know how to make a screenshot but would be happy to learn it.
Thank you

**Comment by 1dog2boys** [comment]
lo5tinpr0
I'm reading about what the snapshot is and seems like there is my problem. The last snapshot was created on January 28th, 2018 and that's fits my activities since from December last year till now I just parked my tokens in the wallet was inactive and of course  I did not upgrade.

**Comment by 1dog2boys** [comment]
It seems that I'm up to something I had before the IOTA.Wallet2.3.0 and not upgraded till now...here is the problem...

**Comment by lo5tinpr0** [comment]
Hey @1dog2boys , every node, which is working is fine for you. So if you are able to log in with your seed, the node will also work so far. Nodes are nothing else than servers that ensure that you can log in with your seed and that transactions goes from A to B. Nodes can be offline, so don't be surprised if you can't log in. Just change the node and then it usually works.

> As of now 96 transfers 86 addresses but interesting the history doesn't go to far the oldest is from 29/04/2018 nothing prior...

The last snapshot took place on April 29, 2018. It is quite normal and okay that there are only transactions in your history up to this point in time. It looks the same for me and that's the way it should be.

> I don't think this how it should operate. What happened with all my transfers when I sent the IOTA tokens from the trade sites to my wallet. I know they arrived I checked them. Also if I go to the trade places and see the history over there I can see the exact time and addresses where the tokens been sent to (my address). So I wonder what's going on?

This is where the last snapshot comes in. IOTA uses snapshots to keep the size of the Tangle as small as possible. You want to avoid that each node needs a lot of storage space. So what happens when a snapshot is taken? A snapshot removes all zero value transactions from the Tangle and deletes them from your history.
Don't worry, no IOTAs will be deleted! All your transactions with value are still there. They are only no longer displayed in the history.

As I said, the last snapshot took place on April 29th. The one before that on January 28th. So if you don't log in with your seed for a longer time, it can look as if everything has suddenly been "deleted": The balance is displayed with 0 and the history is completely empty.
But don't worry! This is just the result of the last snapshot. All your IOTAs are still there.

How to make sure that the balance shows the correct value of IOTAs again?

You've already started with the right thing and created new addresses and attached these to the tangle. Sometimes you have to generate as many new addresses and attach them to the tangle as you already generated before the snapshot. Usually, however, a few new addresses should suffice.
You can find a good explanation here: [https://medium.com/@ralf/prepare-for-the-january-28-2018-iota-snapshot-10f565b371ab](https://medium.com/@ralf/prepare-for-the-january-28-2018-iota-snapshot-10f565b371ab)
Follow the appropriate steps and try to "restore" your balance.

I admit, IOTA and the corresponding wallets may not be easy to use yet, but there will be good solutions for that too in the near future.

How to make a screenshot:
If you use Windows, you can use the so called "snipping tool". You can find this by pressing win + s and typing snipping tool. You can use it to copy your entire screen or parts of your screen and save it as an image. You can then send this picture.
But I think we can do that in this case without screenshots.

Let me know if you were successful and your balance now shows the right value again. If you have any further questions, please do not hesitate to contact us.

**Comment by kondi006** [comment]
Hello,

I have the same problem but with 2.5.7 wallet. I bought my IOTAs in December 2017 and now I don't see them. I can find my balance with tool "balance-finder" but can't make this adress being generated and still has 0. For now i've made over 60 adresses and still don't see my balance like it should be. Balance-finder shows it okay. Is there something else i can do? I don't have all week to make receive and so on. I know exact adress of my wallet connected to seed where my balance is. Can I in some way send it somewhere or make generator check adress which I put in wallet?

**Comment by lo5tinpr0** [comment]
Hey @kondi006 , could you please post the address where your IOTAs should be?
Usually it is enough to generate some addresses. 60 sounds a little much...

Please check out this article and try it with step by step: [https://medium.com/@ralf/prepare-for-the-january-28-2018-iota-snapshot-10f565b371ab](https://medium.com/@ralf/prepare-for-the-january-28-2018-iota-snapshot-10f565b371ab)

**Comment by kondi006** [comment]
Wallet adress: GVZKAVLFRTLQUGOQD9GACSWDLQQGLVJTPYGO9DUTUFPLSEAGENMTBRMGRTSIZJQNYNOQZEWAITNZDKYDX

Balance-finder gives me error: "Deposit adress has been spent from" when I want to send it to new one.

**Comment by lo5tinpr0** [comment]
Hey @kondi006 , this address

> GVZKAVLFRTLQUGOQD9GACSWDLQQGLVJTPYGO9DUTUFPLSEAGENMTBRMGRTSIZJQNYNOQZEWAITNZDKYDX

is empty and the zero balance is shown correct. Have a look up here: [https://thetangle.org/address/GVZKAVLFRTLQUGOQD9GACSWDLQQGLVJTPYGO9DUTUFPLSEAGENMTBRMGRTSIZJQNYNOQZEWAITNZDKYDX](https://thetangle.org/address/GVZKAVLFRTLQUGOQD9GACSWDLQQGLVJTPYGO9DUTUFPLSEAGENMTBRMGRTSIZJQNYNOQZEWAITNZDKYDX)

> Balance-finder gives me error: "Deposit adress has been spent from" when I want to send it to new one.

That's the result of the address reuse protection and how it should be.

**Comment by lo5tinpr0** [comment]
All your IOTAs went to this address: [https://thetangle.org/address/CJRNMIZZUWDNPSHPRLV9NYSEIVJNOJCUPPVFITCOTHEYRSVOQYRUOCGIUAIWDIQHR9AYSYDJZMKYASGDZ](https://thetangle.org/address/CJRNMIZZUWDNPSHPRLV9NYSEIVJNOJCUPPVFITCOTHEYRSVOQYRUOCGIUAIWDIQHR9AYSYDJZMKYASGDZ)

In my opinion, someone had access to your seed and could transfer your IOTAs.
How this could happen, see #1196

**Comment by kondi006** [comment]
No nobody has access to it. But okay. It's my last thing with IOTA. Protection like only seed and dissapearing funds. Sorry but it's not good for crypto like that. I've created my seed using key words, by offline. The transactions are made at night.  I don't know how the hell it could happen...

**Comment by lo5tinpr0** [comment]
> So there is no way to bring them back or to check where they had gone? 

No, there is no chance to reverse transactions at all. You can check the way your IOTAs went by using the link above. That will be a neverending transaction story from wallet to wallet.

> And why balance finder shows there is a balance there?

I don't know. I don't work with the balance finder. The only thing I can tell you is that your balance is zero and technically, it all went well.
You are not the only one who was scammed in January 2018. To be clear, this might be not a real hack with which someone broke the IOTA security, it seems to be a very smart social engeneering hack.
As far as I know, the seeds could get known in any way by someone and were used to transfer IOTAs via several wallets.

**Comment by rajivshah3** [comment]
Hi,
I made IOTA-Balance-Finder and it seems there’s a bug where it says the deposit address has been spent from even if it hasn’t been spent from. I haven’t had time to investigate it though, unfortunately.

**Comment by kondi006** [comment]
Hi,
How your balance-finder works? Because it says my money is at adress connected with my seed and tangle says something else, so what is true? At tangle there is an output transaction from this wallet adress, the same at which balance-finder shows there is my money.

**Comment by 1dog2boys** [comment]
**So now the time arrived to update the info on my quest to regain my IOTAs. 
First I updated my IOTA wallet, generated new addresses 130 times, also changed the nodes several times but nothing the balance still 0.  Then I started to read about the snapshot(s) which happened several times in 22 of Sept. 24thof October, 28th of January and also last week. The most helpful article: https://medium.com/iota-demystified/help-my-iota-balance-is-zero-579b75f35f2c  read it you can find a lot of answers there. I concluded that my IOTAs been taken into custody by the staff to protect me...nice! except if they do that wouldn't be nicer if they notify me and everyone else affected and not just waiting us to get a heart attack when we find out that our  IOTAs went missing?  If you assume you are a victim of the IOTA foundations kindness and  your addresses was moved to the IOTA foundation custody address you can go check it  and make sure on the following link: https://gist.githubusercontent.com/cyclux/2bb05d873c4ec6115cad1d100263d489/raw/612a49e46091586957448f6606ea981ab18022e7/snapshot_validation_20171023.txt
copy, control F, paste ...and I found all my missing (confiscated) IOTAs on the list! But my category was not mentioned in the article which is: curl_not_transitioned. I don't know what to think of this category could not find any info. Anyway  move on now the issue how to get them back!? To the  Reclaim tool...I got ready with the old seed also with the new seed...pretty simple  eventually I got "Proof transaction is pending" and pending and pending time sometimes you prioritize and reattach and again and you have to go to work or to sleep and you  have to turn off you computer but as soon is possible you start again and only two days later "Proof transaction confirmed".  Great wrote down the address it gave me went to the new seed and receive the same address shows up and only a couple hours later confirmed...and still 0. F...ing ZERO.
Went back to check the custody (confiscated)  address and everything is still there unchanged!
SO now what? Really WHAT???
In conclusion: IOTA aims for machine to machine communications...the future. How about to try just plain COMMUNICATIONs first like human to human or IOTA to human! If the future you about to build is unhuman, arrogant, top down like a dictatorship that's not a good future and I don't think that we should support it.**

**Comment by 1dog2boys** [comment]
Thank you

I'll read it

Laszlo


________________________________

**Comment by cryptotom77** [comment]
There is a website which is gathering information from users with stolen Iotas to file a class action suit against that joke of a crypto.  Here is the link:
https://iotawalletloss.claims/
